cypress/e2e/shared/fluxQueryBuilder.test.ts

```diff
@@ -506,7 +517,118 @@ describe('Script Builder', () => {
       }).then(() => {
         clearSession()
         cy.getByTestID('flux-sync--toggle')
-        cy.getByTestID('flux-editor', {timeout: DELAY_FOR_LSP_SERVER_ONLINE})
+        cy.getByTestID('flux-editor', {timeout: DELAY_FOR_LAZY_LOAD_EDITOR})
+      })
+    })
+
+    describe('basic functionality', () => {
+      it('can contruct a composition with fields', () => {
+        cy.log('empty editor text')
+        cy.getByTestID('flux-editor').monacoType('{selectAll}{del}')
+
+        cy.log('Ensure LSP is online') // deflake
+        cy.wait(DELAY_FOR_LSP_SERVER_BOOTUP)
+
+        cy.log('select bucket and measurement')
+        selectSchema()
+        confirmSchemaComposition()
+
+        cy.log('select field --> adds to composition')
+        const fieldName0 = 'station_id_0'
+        const fieldName10 = 'station_id_10'
+        selectListItem(fieldName0, true)
+        cy.getByTestID('flux-editor').contains(
+          `|> filter(fn: (r) => r._field == "${fieldName0}")`
+        )
+        selectListItem(fieldName10, true)
+        cy.getByTestID('flux-editor').contains(
+          `fn: (r) => r._field == "${fieldName0}" or r._field == "${fieldName10}"`
+        )
+
+        cy.log('select field --> removes from composition')
+        selectListItem(fieldName10, false)
+        cy.wait(1000)
+        cy.getByTestID('flux-editor').contains(
+          `|> filter(fn: (r) => r._field == "${fieldName0}")`
+        )
+        cy.getByTestID('flux-editor').within(() => {
+          cy.get('textarea.inputarea').should('not.contain', fieldName10)
+        })
+      })
+
+      it('can contruct a composition with tagValues', () => {
+        cy.log('empty editor text')
+        cy.getByTestID('flux-editor').monacoType('{selectAll}{del}')
+
+        cy.log('Ensure LSP is online') // deflake
+        cy.wait(DELAY_FOR_LSP_SERVER_BOOTUP)
+
+        cy.log('select bucket and measurement')
+        selectSchema()
+        confirmSchemaComposition()
+
+        cy.log('select tagValue --> adds to composition')
+        cy.getByTestID('container-side-bar--tag-keys').within(() => {
+          cy.getByTestID('accordion-header').should('be.visible').click()
+        })
+        const tagKey = 'air_temp_degc'
+        const tagValue = '70_degrees'
+        selectListItem(tagValue, true)
+        cy.getByTestID('flux-editor').contains(
+          `|> filter(fn: (r) => r.${tagKey} == "${tagValue}")`
+        )
+
+        cy.log('select tagValue --> removes from composition')
+        cy.wait(1000)
+        cy.getByTestID('flux-editor').within(() => {
+          cy.get('textarea.inputarea').should('not.contain', tagKey)
+        })
```

Comment:
Just curious how the `tagKey` get removed here? I don't see a `selectListItem(tagValue, false)` here

Reply:
Nice catch.
removing from merge queue, and pushing up the fix.

Reply:
Same. That's why I had the question, and thinking maybe I missed something since the test passed on CI earlier